tests/AeroSharp.Tests/Utility/TestPreparer.cs

```diff
@@ -37,7 +37,7 @@ public ClientWrapper GetClient()
         private const int Port = 3000;
 
         public const string TestSet = "test_set";
-        public const string TestNamespace = "test_namespace";
+        public const string TestNamespace = "test";
```

Comment:
TestNamespace changed because the github action sets up the aerospike instance with a different namespace.